extensions/wikia/FandomComMigration/FandomComMigrationHooks.class.php

```diff
@@ -3,29 +3,19 @@
 class FandomComMigrationHooks {
 
 	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
-		if ( static::isEnabled() ) {
+		if ( static::isMigrationScheduled() ) {
 			$out->addModules( 'ext.fandomComMigration' );
 		}
 
 		return true;
 	}
 
 	public static function onMercuryWikiVariables( array &$wikiVariables ): bool {
-		global $wgFandomComMigrationDone, $wgFandomComMigrationCustomMessageBefore,
```

Comment:
I think Growth is using this for displaying various messages on wikis, for example on https://fishing-simulator.fandom.com/wiki/Fishing_Simulator_Wiki. I think they won't be happy if you remove it